src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,28 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int bladeLength;
```

Comment:
Don't forget about access modifiers for your custom fields in `Bulldozer` class. It's a good practice to make fields private and provide public getter and setter methods for accessing them.